libs/ui-components/src/components/common/TechPreviewBadge.tsx

```diff
@@ -27,18 +27,16 @@ const TechPreviewBadge = () => {
   const { t } = useTranslation();
 
   return (
-    <Label color="orange" className="pf-v5-u-ml-md">
-      <Popover
-        aria-label={t('Technology preview description')}
-        bodyContent={<TechPreviewPopoverContent />}
-        withFocusTrap
-        triggerAction="click"
-      >
-        <span>
-          <InfoCircleIcon /> {t('Technology preview')}
-        </span>
-      </Popover>
-    </Label>
+    <Popover
+      aria-label={t('Technology preview description')}
+      bodyContent={<TechPreviewPopoverContent />}
+      withFocusTrap
+      triggerAction="click"
+    >
+      <Label color="orange" icon={<InfoCircleIcon />}>
```

Comment:
Hi! I was aware of the icon color, but in the screenshot in https://issues.redhat.com/browse/EDM-1099 it was with the black icon. Maybe we could check with Nir, though I also like the orange more.

Reply:
That is actually PF6 styling, in PF5 the icons are in the same color as label.

Reply:
mm actually we should match what ACM do?

Reply:
They just use the `Label` component, `orange` color. No additional styling.

Reply:
Thanks, based on the discussions, your changes are correct.